### PR TITLE
Fix Coding standard and made compatible with magento version ^2.4.5

### PR DESCRIPTION
**Pull request by harshdss**
Fix Coding standard and made compatible with magento version ^2.4.5

### CODE REVIEW

File: app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        return $this->goToCatalogHelper->getUrl('catalogsearch/result', ['_secure' => true]) . '?q=' . $keyword;
+    }
+    
+    /**
+     * Get Parent
+     *
+     * @param int|string $productId
+     * @return void
+     */
+    private function getParent($productId)
```
Comment (harishmk1805) [inline app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php]: return type missing

File: app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        return false;
+    }
+    
+    /**
+     * Get Default Url
+     *
+     * @param string $keyword
+     * @return string
+     */
+    public function getDefaultUrl($keyword): string
```
Comment (harishmk1805) [inline app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php]: type missing

**Review by harishmk1805 — CHANGES_REQUESTED** [review]

File: app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php (hunk 1 above)
Comment (harshdss) [reply app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php]: Please check i added return type.

File: app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php (hunk 2 above)
Comment (harshdss) [reply app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php]: please check i add type

File: app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php [hunk 3]
```diff
@@ -86,9 +86,9 @@ public function getDefaultUrl($keyword): string
      * Get Parent
      *
      * @param int|string $productId
-     * @return void
+     * @return bool|array|string
      */
-    private function getParent($productId)
+    private function getParent($productId): bool|array|string
```
Comment (harishmk1805) [inline app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php]: int $productId

**Review by harishmk1805 — CHANGES_REQUESTED** [review]

File: app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php (hunk 3 above)
Comment (harshdss) [reply app/code/Dss/GoToCatalog/Model/ProductUrlResolver.php]: I update code please check.

**Pull request merged**